packages/mui-base/src/InputUnstyled/InputUnstyled.test.tsx

```diff
@@ -30,4 +33,26 @@ describe('<InputUnstyled />', () => {
 
     expect(screen.getByRole('textbox')).to.have.tagName('textarea');
   });
+
+  describe('componentsProps', () => {
```

Comment:
Should we add this in the conformance tests?

Reply:
Sure, I've just added it.